packages/resolver/src/adapters/handle-api.ts

```diff
@@ -0,0 +1,106 @@
+import {Api, Resolver} from '@yoroi/types'
+import {fetchData, FetchData, handleApiError, isLeft} from '@yoroi/common'
+import {z} from 'zod'
+import {AxiosRequestConfig} from 'axios'
```

Comment:
Should it be wrapped in `@yoroi/common`?

Reply:
yap, is a little bit leaky since is exposing the "fetcher" config directly, but also didn't want a 1:1, it's a @banklesss good call for an improvement, maybe we can expose only what we are actually using.